uber/site_sections/registration.py

```diff
@@ -563,7 +563,14 @@ def printed_badges(self, session, message='', id=None, pending=None, reprint_rea
         else:
             badges = session.query(Attendee).filter(Attendee.status == PRINTED_STATUS).order_by(Attendee.badge_num).all()
 
+        page = int(page)
```

Comment:
This is copy-pasted from the registration index page for convenience, but we may want to consider implementing a server-side pagination system that we can apply to any function.